tests/cross_fw/examples/example_scope.json

```diff
@@ -0,0 +1,147 @@
+            "int8_mAP": 0.4425154975267813,
+            "accuracy_drop": 0.01001205312497122
+        },
+        "performance_metrics": {
+            "fp32_fps": 170.69,
+            "int8_fps": 444.15,
+            "performance_speed_up": 2.6020856523522173
+        }
+    },
+    "post_training_quantization_openvino_anomaly_stfpm_quantize_with_accuracy_control": {
+        "backend": "openvino",
+        "requirements": "examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/requirements.txt",
+        "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "accuracy_metrics": {
+            "fp32_top1": 0.9356223344802856,
+            "int8_top1": 0.9356223344802856,
+            "accuracy_drop": 0.0
+        },
+        "performance_metrics": {
+            "fp32_fps": 331.61,
+            "int8_fps": 937.34,
+            "performance_speed_up": 2.8266336962094027
+        },
+        "model_size_metrics": {
+            "fp32_model_size": 21.33378505706787,
+            "int8_model_size": 5.616154670715332,
+            "model_compression_rate": 3.7986462816471147
+        }
+    },
+    "post_training_quantization_openvino_yolo8_quantize_with_accuracy_control": {
+        "backend": "openvino",
+        "requirements": "examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/requirements.txt",
+        "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "accuracy_metrics": {
+            "fp32_mAP": 0.44964819083938,
+            "int8_mAP": 0.4428344895334926,
+            "accuracy_drop": 0.006813701305887432
+        },
+        "performance_metrics": {
+            "fp32_fps": 134.48,
+            "int8_fps": 274.58,
+            "performance_speed_up": 2.0417906008328375
+        }
+    },
+    "post_training_quantization_tensorflow_mobilenet_v2": {
+        "backend": "tensorflow",
+        "requirements": "examples/post_training_quantization/tensorflow/mobilenet_v2/requirements.txt",
+        "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "accuracy_metrics": {
+            "fp32_top1": 0.987770676612854,
+            "int8_top1": 0.9775795936584473,
+            "accuracy_drop": 0.010191082954406738
+        },
+        "performance_metrics": {
+            "fp32_fps": 1703.04,
+            "int8_fps": 5796.3,
+            "performance_speed_up": 3.403501972942503
+        },
+        "model_size_metrics": {
+            "fp32_model_size": 8.596238136291504,
+            "int8_model_size": 2.69466495513916,
+            "model_compression_rate": 3.1900953474371994
+        }
+    },
+    "post_training_quantization_torch_mobilenet_v2": {
+        "backend": "torch",
+        "requirements": "examples/post_training_quantization/torch/mobilenet_v2/requirements.txt",
+        "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "accuracy_metrics": {
+            "fp32_top1": 0.8761783439490446,
```

Comment:
I got a different resulting metric when running this test:
```
>           assert measured_metrics[name] == pytest.approx(value, abs=ACCURACY_TOLERANCE)
E           assert 0.9864968152866243 == 0.8761783439490446 ± 2.0e-03
E             comparison failed
E             Obtained: 0.9864968152866243
E             Expected: 0.8761783439490446 ± 2.0e-03
```

Could you please check again?

Reply:
Surprised, Could you share how you run test? I used `make test-examples`.
[log.txt](https://github.com/openvinotoolkit/nncf/files/11757668/log.txt)

Reply:
I run it via `PYTHONPATH=../../.. pytest . -s`
But I get the same failure when running according to your command: [test_examples_log.txt](https://github.com/openvinotoolkit/nncf/files/11760272/test_examples_log.txt)

I see your run takes almost two hours, while mine is only half an hour. Do you run on a CPU?

Reply:
Thanks, updated.